src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
* Kindly use the [ducks' pattern](https://github.com/erikras/ducks-modular-redux) to write the values of your action types ( ie it should look something like this `const ADD_BOOK = bookstore/books/ADD_BOOK` ) as it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Write%20your%20book%27s%20actions%20and%20reducer%20(in%20/src/redux/books/books.js)%20using%20the%20ducks%20pattern.)